### PR TITLE
feat: create shipping address model

### PR DESCRIPTION
**Pull request by tee-code**
Kindly review the PR.

### CODE REVIEW

**Comment by tee-code** [comment]
I followed the data types used on Customer model created by Mary which is slightly different from the diagram. Let me know if am to switch back to the diagram. 

Note that the customer and order relationships would be created on their models as well. 

Thanks.

**Comment by samdoghor** [comment]
Did you test run to ensure the database is working fine? If you did, you didn't commit your migration file.

**Review by samdoghor — CHANGES_REQUESTED** [review]
Tables names are now agreed to be in snake-case and not camel-case.

**Comment by tee-code** [comment]
I do not want create migration file when the model is not approved. Just go through it for other corrections apart from table name and I will fix all at once. Thanks

**Comment by samdoghor** [comment]
Everything seems ok, except the phone number and zipcode, any specific reason why you are using string as data type.

File: src/models/shipping_address.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from .abc import BaseModel, MetaBaseModel
+from datetime import datetime
+
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class ShippingAddress(db.Model, BaseModel, metaclass=MetaBaseModel):
+    """ The Shipping Address model """
+
+    __tablename__ = "shippingAddresses"
```
Comment (samdoghor) [inline src/models/shipping_address.py:14]: Table name should be in snake case

File: src/models/shipping_address.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    id = db.Column(db.Integer, primary_key=True)
+    country = db.Column(db.String(50), nullable=False)
+    state = db.Column(db.String(70), nullable=False)
+    city = db.Column(db.String(50), nullable=False)
+    street_name = db.Column(db.String(300), nullable=False)
+    zipcode = db.Column(db.String(50))
```
Comment (samdoghor) [inline src/models/shipping_address.py:21]: This is ok, do not change the data type.

File: src/models/shipping_address.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    phone = db.Column(db.String(15), nullable=False)
```
Comment (samdoghor) [inline src/models/shipping_address.py:22]: This is ok, do not change the data type.

**Review by samdoghor — CHANGES_REQUESTED** [review]

**Pull request closed without merging**